### PR TITLE
Update routes.php

### PR DESCRIPTION
**Pull request by phixxx5**
Added 'Lineare Algebra'

### CODE REVIEW

File: redirect/routes.php [hunk 1]
```diff
@@ -178,6 +178,10 @@ class Route {
             'description' => 'Grundlegende Algorithmen und Datenstrukturen',
             'target'      => 'http://campar.in.tum.de/Chair/TeachingSs18GAD',
         ],
+        'linalg'          => [
```
Comment (rudxde) [inline redirect/routes.php]: the tag for 'Lineare Algebra für Informatik' is 'LA'

Comment (kordianbruck) [reply redirect/routes.php]: What exactly do you mean? Are you suggesting an additional synonym?

Comment (rudxde) [reply redirect/routes.php]: i don't know if it is possible otherwise i suggest to change the tag

Comment (kordianbruck) [reply redirect/routes.php]: Sure is, checkout the synonyms array: https://github.com/phixxx5/TUM.sexy/blob/34391c0c78e5f87f0118833a4c367abf8c887213/redirect/routes.php#L255

Feel free to open a Pull Request for your suggestion :+1:

**Pull request merged**